packages/language-server/server/validation/type_definitions.js

```diff
@@ -0,0 +1,112 @@
+const path = require("path");
+const {
+  promises: { readFile },
+} = require("fs");
+const babylon = require("@babel/parser");
+const { createModuleScope } = require("@hegel/core/type-graph/type-graph");
+
+const { dtsrc } = require("../parser_settings");
+
+let stdLibTypeGraph;
+let browserGlobalTypeGraph;
+let nodeJsGlobalTypeGraph;
+
+function mixTypeDefinitions(config) {
+  return async (globalScope) => {
+    if (stdLibTypeGraph === undefined) {
+      stdLibTypeGraph = await getStandardTypeDefinitions(globalScope);
+    }
+    mixSomeTypeDefinitions(globalScope, stdLibTypeGraph);
+
+    /**
+     * By default Hegel will think that code is acceptable for both
+     * platforms: "nodejs" and "browser". If other options is not set in config file.
+     */
+    const shouldIncludeNodeJS = config.environment
+      ? config.environment.includes("nodejs")
+      : true;
```

Comment:
Here maybe should be `false` if environment is not defined. I just thought that it would be useful to have full code checking and completion by default.

Reply:
It should be `false` because we can't detect without configuration is the project for `nodejs` or for `browser`. So, it will be a problem if by default we will use `nodejs` in the `browser` project.